testing/trino-product-tests/pom.xml

```diff
@@ -385,20 +385,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
```

Comment:
there were exclusions in pinot and ES as well IIRC. Are they safe to remove now or does ES client pull it's own jackson?

Reply:
They are pulling snakeyaml through different dependencies